src/sentry/api/endpoints/group_integration_details.py

```diff
@@ -4,17 +4,34 @@
 
 from rest_framework.response import Response
 
-from sentry import analytics
+from sentry import analytics, features
 from sentry.api.bases import GroupEndpoint
 from sentry.api.serializers import serialize
 from sentry.api.serializers.models.integration import IntegrationIssueConfigSerializer
 from sentry.integrations import IntegrationFeatures
 from sentry.integrations.exceptions import IntegrationError, IntegrationFormError
 from sentry.models import ExternalIssue, GroupLink, Integration
 
+MISSING_FEATURE_MESSAGE = 'Your organization does not have access to this feature.'
+
 
 class GroupIntegrationDetailsEndpoint(GroupEndpoint):
+    def _has_issue_feature(self, organization, user):
+        has_issue_basic = features.has('organizations:integration:issue_basic',
```

Comment:
Let's update it to `organizations:integrations-issue-basic` and `organizations:integrations-issue-sync`